src/clients/keycloak.ts

```diff
@@ -283,31 +272,20 @@ export function createKeycloakClient(): Client {
     return initPromise;
   };
 
-  const getOrLoadUser: Client['getOrLoadUser'] = () => {
-    const currentUser = getUser();
-    if (currentUser) {
-      return Promise.resolve(currentUser);
-    }
-    if (isInitialized()) {
-      return Promise.resolve(undefined);
-    }
-    return new Promise((resolve, reject) => {
-      init()
-        .then(() => resolve(getUser()))
-        .catch(e => reject(e));
-    });
-  };
+  const getOrLoadUser = createClientGetOrLoadUserFunction({
+    getUser,
+    isInitialized,
+    init
+  });
 
   const getApiAccessToken: Client['getApiAccessToken'] = async (
     options: FetchApiTokenOptions
-  ) => {
-    const tokenResponse = await fetchApiToken({
+  ) =>
+    fetchApiToken({
       uri: getTokenUri(getClientConfig()),
       accessToken: keycloak.token as string,
       ...options
     });
-    return tokenResponse;
-  };
```

Comment:
This doesn't contain an `await` anymore. Is that alright?

Reply:
"await" is redundant on a return value of an async function. It will return a Promise and therefore return value is awaited where function is called. 

Trying to use await there also causes linting error.